components/automate-deployment/pkg/client/automate_config.go

```diff
@@ -14,6 +16,25 @@ import (
 	"github.com/chef/automate/components/automate-deployment/pkg/cli"
 )
 
+var backupPathFix = `sleep 5
+chef-automate status --wait-for-healthy > /dev/null
+indices=(
+chef-automate-es6-automate-cs-oc-erchef
+chef-automate-es6-compliance-service
+chef-automate-es6-event-feed-service
+chef-automate-es6-ingest-service
+)
+for index in ${indices[@]}; do
+curl -XPUT -k -H 'Content-Type: application/json' http://localhost:10144/_snapshot/$index --data-binary @- << EOF
```

Comment:
Have you validated for external open search as well?

Reply:
yes